packages/clerk-js/src/core/resources/Client.ts

```diff
@@ -76,13 +78,30 @@ export class Client extends BaseResource implements ClientResource {
     return this.sessions.forEach(s => s.clearCache());
   }
 
+  // isEligibleForTouch returns true if the client cookie is due to expire in 8 days or less
+  isEligibleForTouch(): boolean {
```

Comment:
The method is used in `packages/clerk-js/src/core/clerk.ts` to determine if the user needs to be redirected so we would have to keep it public so that it's accessible.